modules/dxp/apps/commerce/commerce-organization-web/src/main/resources/META-INF/resources/js/utils/index.js

```diff
@@ -216,26 +216,21 @@ export function getChartNodeId(data) {
 }
 
 export function formatRootData(rootData) {
-	if (!Array.isArray(rootData)) {
-		rootData = [rootData];
-	}
+	const fakeRoot = {
```

Comment:
There were cases where rootData was not an array. Is it now consistent everywhere?

Reply:
Yes.